src/sql/store.py

```diff
@@ -91,6 +92,7 @@ def store(self, key, query, with_=None):
                 f"Script name ({key!r}) cannot appear in with_ argument"
             )
 
+        query = sqlparse.format(query, strip_comments=True)
```

Comment:
add some comment explaining why we're removing comments

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/894/files#diff-a3d1291b54f0d59f191101a81a116bad5859a2bd81603b9b7c1cbea77d18cc10R94-R97)